src/cpp/fastdds/domain/DomainParticipantFactory.cpp

```diff
@@ -426,6 +426,18 @@ void DomainParticipantFactory::participant_has_been_deleted(
     }
 }
 
+const std::string& DomainParticipantFactory::load_statistics_profiles_from_env()
```

Comment:
If my first comment is addressed this could be changed like this:

```suggestion
static void load_statistics_profiles_from_env(
        std::string& topics)
```